prow/test_debug.md

```diff
@@ -0,0 +1,77 @@
+# Testing and Debugging Prow
+
+## How to test prow
+
+Build with:
+```
+bazel build //prow/...
+```
+Test with:
+```
+bazel test --features=race //prow/...
+```
+
+**TODO**(spxtr): Unify and document how to run prow components locally.
```

Comment:
maybe time to change spxtr to other people? Also we have improved our local runs for deck & hook maybe we should actually write something up..

Reply:
I switched the assignment to myself. I agree that we should add some docs about local testing, but I don't want to include that in the scope of this PR.